src/DashboardHub/Bundle/AppBundle/Resources/config/routing.yml

```diff
@@ -2,6 +2,10 @@ dashboardhub_app_homepage:
     path:     /
     defaults: { _controller: DashboardHubAppBundle:Default:index }
 
+dashboardhub_app_dashboard:
+    path:     /d/{uid}
```

Comment:
This would become:

`dashboardhub.io/dashboard/{key}`
and
`dashboardhub.io/dashboards/`
`dashboardhub.io/dashboards/add`
`dashboardhub.io/dashboards/edit`

You don't think it is too long & the word `dashboard` repeated too many times?